speculos.py

```diff
@@ -236,16 +238,25 @@ def setup_logging(args):
         }
         args.sdk = default_sdk.get(args.model)
 
+    api_enabled = (args.api_port != 0)
+
     automation_path = None
     if args.automation:
+        logger.warn("--automation is deprecated, please use the REST API instead")
         automation_path = automation.Automation(args.automation)
 
     automation_server = None
     if args.automation_port:
+        logger.warn("--automation-port is deprecated, please use the REST API instead")
+        logger.warn("--automation-port is incompatible with the the API server, disabling it")
+        api_enabled = False
```

Comment:
It seems strange to have two warnings. Is a `if api_enabled:` missing here?

Reply:
Actually it was intended, but let's add this condition